src/Hyperion.Tests/CrossFrameworkSerializationTests.cs

```diff
@@ -19,9 +21,21 @@ public class CrossFrameworkSerializationTests
         public CrossFrameworkSerializationTests(ITestOutputHelper log)
         {
             _log = log;
-            _serializer = new Serializer();
             _originalObject = CrossFrameworkInitializer.Init();
             _originalMixedObject = CrossFrameworkInitializer.InitMixed();
+
+            // Demonstrating the use of custom dll package name override
+            // to convert netcore System.Drawing.Primitives to netfx 
+            // System.Drawing package.
+            #if NETFX
+            _serializer = new Serializer(new SerializerOptions(
+                packageNameOverrides: new List<CrossPlatformPackageNameOverride>
+                {
+                    new CrossPlatformPackageNameOverride("System.Drawing.Primitives", ".Primitives", "")
```

Comment:
So how does this work exactly @Arkatufus ? What's the formula for substituting names in one package for another?

Reply:
The first string is the "fingerprint", the unique string that indicates the wrong package name. It is matched using `string.Contain`.
The second string is the string pattern that needs to be replaced.
The third is the string to replace it with.

In this example, "if the package name contains the string `System.Drawing.Primitives`, replace `.Primitives` with an empty string.

Reply:
How could we make that more intuitive?

Reply:
I don't know, any ideas?

Reply:
Think passing in a `Func<string,string>` predicate would make more sense? Leave it up to the user how to transform the name?

Reply:
Sounds good, I'll make the changes

Reply:
oh... I just remembered... we can't do that, it has to be a string, because this will have to be configurable from hocon

Reply:
We can always pass in a `Setup` for this for Hyperion just like we did for Newtonsoft.Json .

Reply:
And you can add a layer that translates HOCON into these delegates too but for this level of customization passing in an array of typed functions is probably better.